wordle-web/src/views/WordleView.vue

```diff
@@ -17,21 +56,102 @@ import { ref, reactive, onMounted, onUnmounted } from 'vue'
 import GameBoard from '../components/GameBoard.vue'
 import KeyBoard from '../components/KeyBoard.vue'
 import type { Letter } from '@/scripts/letter'
+import Axios from 'axios'
 
 const guess = ref('')
 const game = reactive(new WordleGame())
 
+const overlay = ref(true)
+let nameDialog = ref(false)
+let endScreen = ref(false)
+let displayName = ref(game.displayName)
+let curStatus = game.status
+
 onMounted(async () => {
   window.addEventListener('keyup', keyPress)
-  await game.restartGame()
-  console.log(game.secretWord)
 })
 onUnmounted(() => {
   window.removeEventListener('keyup', keyPress)
 })
 
+function nameUpdater(newName: string) {
+  console.log(newName)
+  let str = newName
+  game.changeName(str)
+  displayName.value = game.displayName
+}
+
+function handleSubmit() {
+  submitToLeaderboard()
+  endScreen.value = false
+  game.restartGame('defal')
+}
+
+async function submitToLeaderboard() {
+  console.log('PlayerInfo')
+  console.log(game.displayName)
+  console.log(game.finalGuesses)
+  console.log(game.finalTime)
+  overlay.value = true
+  await Axios.post(
+    'Leaderboard/InsertScore?name='
+      .concat(game.displayName)
+      .concat('&numAttempts=')
+      .concat(String(game.finalGuesses))
+      .concat('&seconds=')
+      .concat(String(game.finalTime)),
+    {
+      //for some reason, these aren't working correctly, will investigate later
+      /*name: game.displayName,
+      numAttempts: game.finalGuesses,
+      seconds: game.finalTime*/
+    }
+  )
```

Comment:
So in this method, if you modify the AddScore method in the backend, you should be able to send the json object.

Reply:
Alright, I'll get that changed and make sure it works.